src/types/formatted/administration.ts

```diff
@@ -1,4 +1,4 @@
-export type Administration = {
-  id: number
-  name: string
+export interface Administration {
+  entityId: number
```

Comment:
```suggestion
  entityId: number | null
```

It _shouldn't_ be, but our formatter is returning `null` for this in some cases. 🙈

Reply:
I also recommend adding a `/**` comment on the property to describe _which_ ID this is, since there's some confusion around that. 😅 It should be clear when you read the `formatAdministration` function, but this static type definition and that function are far enough removed from each other to make it a cognitive leap to even _know_ you should look there, y'know?